ts/input/tex/FilterUtil.ts

```diff
@@ -225,6 +225,47 @@ namespace FilterUtil {
   };
 
 
+  /**
+   * Looks through the list of munderover elements for ones that have
+   * movablelimits and bases that are not mo's, and creates new msubsup
+   * elements to replace them if they aren't in displaystyle.
+   *
+   * @param {MmlNode} ath The node to rewrite.
+   * @param {ParseOptions} data The parse options.
+   */
+  let _moveLimits = function (options: ParseOptions, underover: string, subsup: string) {
+    for (const mml of options.getList(underover)) {
+      const display = mml.attributes.get('displaystyle');
+      if (!display) {
```

Comment:
Can we have a 
```suggestion
      if (display) {
         continue;
      } 
```
here? It is already very nested.

Reply:
OK, sure.